### ISSUE TITLE
Nidhogg - Dragon Breath Damage

### ISSUE THREAD

**Issue by panfunk** [EdenServer/community#1833]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

While fighting Nidhogg I noticed that his breath attack never did any damage to me. I was hit several times for 0 damage. I didn't have Stoneskin or Rampart buffs, only my -10% Breath damage Hrotti and the usual buffs. (Pro/Shell, songs etc)

**Comment by Slamm21** [comment]
Did you have Phalanx2? If so it could have been giving very low damage but not 0

**Comment by panfunk** [comment]
No, I didn't have Phalanx2, and it would not nullify the average 300-400(? afair) damage breaths.
Also the day before on Fafnir I took the usual damage on breaths.

**Comment by panfunk** [comment]
Just spoke with Cheese, who tanked the Nid before and a recent Faf. He experienced the same:
Fafnir's breath attacks deal 300-400 damage,
Nidhoggs deal way less, even 0 damage.
And yes, we're tanking on their paws.
